draft-ietf-quic-v2.md

```diff
@@ -140,8 +140,17 @@ Compatible version negotiation between versions 1 and 2 follow the same
 requirements in either direction. This section uses the terms "original
 version" and "negotiated version" from {{QUIC-VN}}.
 
-If the server elects to send a Retry packet, it MUST do so using the original
-version.
+If the server sends a Retry packet, it MUST use the original version. The client
+MUST NOT use a different version in the subsequent Initial that contains the
+Retry token. The server MAY encode the QUIC version in its Retry token to
+validate that the client did not switch versions, and drop the packet if it
+switched.
```

Comment:
As noted, this is probably not really necessary, but it's fine.

Reply:
yep